crates/biome_analyze/CONTRIBUTING.md

```diff
@@ -649,6 +659,16 @@ This is what the `Deserializable` keyword in the `#[derive]` statements above di
 It's a so-called derive macros, which generates the implementation for the `Deserializable` trait
 for you.
 
+The rule's options type have also to implement `biome_deserialize::Merge`.
+This allows merging the rule's options with the inherited options when a user's configuration file
```

Comment:
Let's use the term **extended** instead of inherited. We never use that term, plus "extended" is closer to `extends`